apps/frontend-pwa/src/components/CourseElement.tsx

```diff
@@ -0,0 +1,79 @@
+import { faBellSlash } from '@fortawesome/free-regular-svg-icons'
+import { faBell } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import * as TogglePrimitive from '@radix-ui/react-toggle'
+import { Dispatch, SetStateAction, useState } from 'react'
+import { twMerge } from 'tailwind-merge'
+
+type courseElementProps = {
+  courseName: string
+  courseId: string
+  disabled: boolean
+  onSubscribeClick: (
+    subscribed: boolean,
+    setSubscribed: Dispatch<SetStateAction<boolean>>,
+    courseId: string
+  ) => void
+}
+
+const CourseElement = (props: courseElementProps) => {
+  // TODO: Fetch initial subscription state from DB
+  const [subscribed, setSubscribed] = useState(false)
+
+  const computedClassNameSubscribed = twMerge(
+    props.disabled
+      ? 'text-uzh-grey-20'
+      : 'text-uzh-yellow-100 hover:text-uzh-grey-20'
+  )
+
+  const computedClassNameNotSubscribed = twMerge(
+    props.disabled
+      ? 'text-uzh-grey-20'
+      : 'text-uzh-grey-20 hover:text-uzh-yellow-100'
+  )
+
+  const computedClassNameButton = twMerge(
+    'h-full p-3 text-2xl rounded-r-md',
+    props.disabled ? 'bg-slate-400' : 'bg-slate-600'
+  )
+
+  return (
+    <div className="flex justify-between w-full mt-4 mb-4 mr-4align-center">
+      <span className="w-full p-4 text-l text-start rounded-l-md bg-uzh-grey-20 hover:bg-uzh-grey-40">
+        {/* TODO: ev. turn into link */}
+        {props.courseName}
+      </span>
+      <span className={computedClassNameButton}>
+        <TogglePrimitive.Root
+          defaultPressed={subscribed}
+          onPressedChange={setSubscribed}
+          asChild
+        >
+          <button
```

Comment:
I'm not sure if this is a good idea since these are toggle buttons and are styled completely different than the design system button. When I just replace the button and leave the rest as is, this is what it would look like 
![image](https://user-images.githubusercontent.com/58864918/190160727-eef3f983-4c0d-4660-8bb9-4080b3fbc974.png)

while it should look like this: 
![image](https://user-images.githubusercontent.com/58864918/190161189-9303c19b-2558-41cd-80f8-45d8a78424d5.png)